docs/sources/mimir/visualize/native-histograms/_index.md

```diff
@@ -104,10 +111,18 @@ In that case, the current value is an accumulated value over the lifespan of the
 
 ## Create Grafana dashboards
 
-When creating a Grafana dashboard for your native histogram, the most relevant panel types are [Histogram](/docs/grafana/latest/panels-visualizations/visualizations/histogram/) and [Heatmap](/docs/grafana/latest/panels-visualizations/visualizations/heatmap/).
+The panel types [Histogram](/docs/grafana/latest/panels-visualizations/visualizations/histogram/) and [Heatmap](/docs/grafana/latest/panels-visualizations/visualizations/heatmap/) are compatible with the new native histogram data type. Use these to visualize a query such as:
+
+```PromQL
+sum(rate(request_duration_seconds[$__rate_interval]))
+```
+
+For all other panel types, for example [Timeseries](/docs/grafana/latest/panels-visualizations/visualizations/time-series/), use one of the histogram functions above in [Prometheus Query Language](#prometheus-query-language) to visualize a derived float time series.
+
+## Grafana classic explore
 
-In [Explore](https://grafana.com/docs/grafana/latest/explore/), the functions `histogram_count`, `histogram_sum`, and `histogram_quantile` will result in normal floating point series that you can plot as usual.
+In [Explore](https://grafana.com/docs/grafana/latest/explore/), use one of the histogram functions above in [Prometheus Query Language](#prometheus-query-language) to visualize a derived float time series.
 
 {{% admonition type="note" %}}
-Visualizing native histogram series directly in the **Explore** view is a work in progress.
+Visualizing native histogram data type directly without the histogram functions in the **Explore** view is a work in progress.
```

Comment:
Is it `native histogram` or `native histograms`? We need to be consistent throughout the doc.

Reply:
Rather than saying it's a work in progress, we should say it's not supported at this time.